base/inference.jl

```diff
@@ -3409,13 +3408,13 @@ end
 # make sure that typeinf is executed before turning on typeinf_ext
 # this ensures that typeinf_ext doesn't recurse before it can add the item to the workq
 
-precompile(methods(typeinf_edge).defs.sig)
+precompile(typeof(typeinf_edge).name.mt.defs.sig)
```

Comment:
doesn't matter now, but eventually we should delete this line (it's been a no-op ever since inference0.ji was eliminated)